### PR TITLE
[terra-clinical-label-value-view] Remove Terra.it helpers from wdio tests

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary

Remove Terra.it helpers from wdio tests

### CODE REVIEW

**Review by StephenEsser — APPROVED** [review]

File: packages/terra-clinical-label-value-view/tests/wdio/label-value-view-spec.js [hunk 1]
```diff
@@ -1,33 +1,25 @@
 Terra.describeViewports('Label Value View', ['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], () => {
-  describe('when no value input is provided', () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-clinical-label-value-view/clinical-label-value-view/default-label-value-view');
-    });
+  it('when no value input is provided', () => {
+    browser.url('/#/raw/tests/terra-clinical-label-value-view/clinical-label-value-view/default-label-value-view');
```
Comment (mjhenkes) [inline packages/terra-clinical-label-value-view/tests/wdio/label-value-view-spec.js:3]: as part of this please remove all '/#' from the urls, it's just an un-needed redirect at this point.

Comment (mjhenkes) [reply packages/terra-clinical-label-value-view/tests/wdio/label-value-view-spec.js:3]:
```suggestion
    browser.url('/raw/tests/terra-clinical-label-value-view/clinical-label-value-view/default-label-value-view');
```

File: packages/terra-clinical-label-value-view/tests/wdio/label-value-view-spec.js [hunk 2]
```diff
@@ -1,33 +1,25 @@
...
-    Terra.it.validatesElement();
+    Terra.validates.element('no value');
   });
 
-  describe('when a text input is provided', () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-clinical-label-value-view/clinical-label-value-view/text-value-label-value-view');
-    });
+  it('when a text input is provided', () => {
+    browser.url('/#/raw/tests/terra-clinical-label-value-view/clinical-label-value-view/text-value-label-value-view');
```
Comment (mjhenkes) [inline packages/terra-clinical-label-value-view/tests/wdio/label-value-view-spec.js:9]:
```suggestion
    browser.url('/raw/tests/terra-clinical-label-value-view/clinical-label-value-view/text-value-label-value-view');
```

File: packages/terra-clinical-label-value-view/tests/wdio/label-value-view-spec.js [hunk 3]
```diff
@@ -1,33 +1,25 @@
...
-    Terra.it.validatesElement();
+    Terra.validates.element('text value');
   });
 
-  describe('when a node input is provided', () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-clinical-label-value-view/clinical-label-value-view/node-value-label-value-view');
-    });
+  it('when a node input is provided', () => {
+    browser.url('/#/raw/tests/terra-clinical-label-value-view/clinical-label-value-view/node-value-label-value-view');
```
Comment (mjhenkes) [inline packages/terra-clinical-label-value-view/tests/wdio/label-value-view-spec.js:15]:
```suggestion
    browser.url('/raw/tests/terra-clinical-label-value-view/clinical-label-value-view/node-value-label-value-view');
```

File: packages/terra-clinical-label-value-view/tests/wdio/label-value-view-spec.js [hunk 4]
```diff
@@ -1,33 +1,25 @@
...
-    Terra.it.validatesElement();
+    Terra.validates.element('node value');
   });
 
-  describe('when text and node inputs are provided', () => {
-    before(() => {
-      browser.url('/#/raw/tests/terra-clinical-label-value-view/clinical-label-value-view/multiple-value-label-value-view');
-    });
+  it('when text and node inputs are provided', () => {
+    browser.url('/#/raw/tests/terra-clinical-label-value-view/clinical-label-value-view/multiple-value-label-value-view');
```
Comment (mjhenkes) [inline packages/terra-clinical-label-value-view/tests/wdio/label-value-view-spec.js:21]:
```suggestion
    browser.url('/raw/tests/terra-clinical-label-value-view/clinical-label-value-view/multiple-value-label-value-view');
```

**Review by mjhenkes — APPROVED** [review]

**Pull request merged**